package.json

```diff
@@ -534,6 +534,8 @@
     "./RichTextEditor": "./RichTextEditor.js",
     "./Scroller": "./Scroller.js",
     "./Select": "./Select.js",
+    "./SideNav": "./SideNav.js",
+    "./SideNavItem": "./SideNavItem.js",
```

Comment:
nit: this seems like an unrelated change

Reply:
That happened automatically.